### PR TITLE
Version 2.14

### PR DESCRIPTION
**Pull request by larskuhtz**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,19 @@
 # `chainweb-node` Changelog
 
+## 2.14 (2022-04-11)
+
+This version replaces all previous versions. Any prior version will stop working
+on **2022-04-21T00:00:00Z**. Node administrators must upgrade to this version
+before that date.
+
+This version will stop working on **2022-06-16T00:00:00Z**.
```
Comment (mightybyte) [inline CHANGELOG.md:9]: Date looks good.

File: node/ChainwebNode.hs [hunk 2]
```diff
@@ -466,10 +467,10 @@ pkgInfoScopes =
 -- -------------------------------------------------------------------------- --
 -- main
 
--- SERVICE DATE for version 2.13
+-- SERVICE DATE for version 2.14
 --
 serviceDate :: Maybe String
-serviceDate = Just "2022-04-21T00:00:00Z"
+serviceDate = Just "2022-06-16T00:00:00Z"
```
Comment (mightybyte) [inline node/ChainwebNode.hs:473]: Date looks good.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,37 @@
...
+
+*   Pact Changes (#1382, #1396, #1397, #1399, #1407, #1409,#1410, #1413):
+
+    *   Gas changes for integer and decimal operations.
+    *   `NaN` and `+/- Infinity` throw errors now.
+    *   Several other nonsensical arithmetic expressions and operations throw
+        errors now.
+    *   Support of nested Defpacts and native `continue`.
+    *   New natives `create-principal` and `validate-principal`
+    *   Add support for `k:` , `r:`, `m:`, `u:`, `p:`, and `w:` protocol
```
Comment (sirlensalot) [inline CHANGELOG.md:25]:
```suggestion
    *   Add support for principals`r:`, `m:`, `u:`, `p:`, and `w:` in `coin`.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,37 @@
...
+        validation in `coin`.
```
Comment (sirlensalot) [inline CHANGELOG.md:26]:
```suggestion

```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,37 @@
...
+    *   Addition of `fungible-xchain-v1` interface, which provides a
+        `TRANSFER-XCHAIN` managed capability, as well as a
+        `TRANSFER-XCHAIN-RECD` event for general use.
```
Comment (sirlensalot) [inline CHANGELOG.md:29]:
```suggestion
        `TRANSFER-XCHAIN-RECD` event for recording crosschain transfer receipt.
```

**Review by sirlensalot — APPROVED** [review]
Looks good. Have we reviewed/double checked fork dates?

**Comment by larskuhtz** [comment]
> Looks good. Have we reviewed/double checked fork dates?

👍 

image

**Review by edmundnoble — APPROVED** [review]

**Pull request merged**